src/metax_api/models/catalog_record.py

```diff
@@ -1195,6 +1199,13 @@ def _pre_create_operations(self, pid_type=None):
 
             self.date_cumulation_started = self.date_created
 
+        if (self.catalog_is_ida() or self.catalog_is_att()):
```

Comment:
If I understood correctly, the issued should be auto-filled for every dataset except for harvested?

Reply:
Good point, now i modified it to be mandatory for all except harvested and drafts...